src/main/java/storage/Storage.java

```diff
@@ -17,7 +17,7 @@
 
     private static final String LIST_FILE_PATH = "./data/duke.txt";
 
-    public List<Task> load() {
+    public static List<Task> load() {
```

Comment:
Do you think it will be clearer if your load method follows the name of form loadSomething?